src/dashboard/src/components/administration/forms.py

```diff
@@ -258,6 +265,20 @@ class ProcessingConfigurationForm(forms.Form):
         'label': 'Approve normalization',
         'yes_option': '1e0df175-d56d-450d-8bee-7df1dc7ae815',
     }
+    processing_fields['153c5f41-3cfb-47ba-9150-2dd44ebc27df'] = {
+        'type': 'boolean',
+        'name': 'policy_checks_preservation_derivatives',
+        'label': 'Perform policy checks on preservation derivatives',
+        'yes_option': '3a55f688-eca3-4ebc-a012-4ce68290e7b0',
+        'no_option': 'b7ce05f0-9d94-4b3e-86cc-d4b2c6dba546',
+    }
+    processing_fields['8ce07e94-6130-4987-96f0-2399ad45c5c2'] = {
+        'type': 'boolean',
+        'name': 'policy_checks_access_derivatives',
+        'label': 'Perform policy checks on access derivatives',
```

Comment:
This proposed change seems confusing to me since this decision point governs whether "policyCheckAccessDerivative.py" gets called and there is a distinct client script named "validateAccessDerivative.py" that has nothing to do with this...